RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD.cc

```diff
@@ -75,14 +75,14 @@ class rerunMVAIsolationOnMiniAOD : public edm::one::EDAnalyzer<edm::one::SharedR
       TauIdMVAAuxiliaries clusterVariables_;
 
       edm::EDGetTokenT<pat::TauCollection> tauToken_;
-      edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationToken_;
+    /*edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationVLooseToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationLooseToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationMediumToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationTightToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationVTightToken_;
       edm::EDGetTokenT<pat::PATTauDiscriminator> mvaIsolationVVTightToken_;
-      edm::EDGetTokenT<pat::PATTauDiscriminator> mvaEleRawToken_;
+      edm::EDGetTokenT<pat::PATTauDiscriminator> mvaEleRawToken_;*/
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant